sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -130,9 +130,9 @@ The Text Analytics client library provides a [TextAnalyticsClient][text_analytic
 asynchronous operations to access a specific use of Text Analytics, such as language detection or key phrase extraction.
 
 ### Text input
-A **text input**, sometimes called a **document**, is a single unit of input to be analyzed by the predictive models
-in the Text Analytics service. Operations on Text Analytics client may take a single text input or a collection
-of inputs to be analyzed as a batch. 
+A **text input**, sometimes called a **document**, is a single unit of document to be analyzed by the predictive models
```

Comment:
I think it's better to say "also called a document" instead

Reply:
You could also do:
~~A **text input**, sometimes called~~ A document is a single unit of input to be analyzed....